.github/workflows/ai-code-review.yml

```
@@ -0,0 +1,320 @@
+      findings-count: ${{ steps.security.outputs.findings-count }}
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Get PR diff
+        id: diff
+        env:
+          GH_TOKEN: ${{ github.token }}
+        run: |
```

actionlint (1.7.8)

shellcheck reported issue in this script: SC2086:info:4:27: Double quote to prevent globbing and word splitting

(shellcheck)